mne/preprocessing/_regress.py

```diff
@@ -4,28 +4,44 @@
 
 import numpy as np
 
+from ..defaults import _BORDER_DEFAULT
 from ..epochs import BaseEpochs
-from ..io.pick import _picks_to_idx
+from ..io.pick import _picks_to_idx, pick_info
 from ..io.base import BaseRaw
-from ..utils import _check_preload, _validate_type, _check_option, verbose
+from ..utils import (_check_preload, _validate_type, _check_option, verbose,
+                     fill_doc, copy_function_doc_to_method_doc)
+from ..minimum_norm.inverse import _needs_eeg_average_ref_proj
+from ..viz import plot_regression_weights
+from .. import Evoked
 
 
 @verbose
-def regress_artifact(inst, picks=None, picks_artifact='eog', betas=None,
-                     copy=True, verbose=None):
+def regress_artifact(inst, picks=None, exclude='bads', picks_artifact='eog',
```

Comment:
For backward compat here (might not be `flake8`-compat yet) you need the `*`
```suggestion
def regress_artifact(inst, picks=None, *, exclude='bads', picks_artifact='eog',
```